python/fledge/services/core/server.py

```diff
@@ -895,7 +895,28 @@ def _start_core(cls, loop=None):
                                  "Therefore, starting the service...")
                     loop.run_until_complete(cls.add_and_enable_dispatcher())
                     _logger.info("Dispatcher service started.")
-
+                schedule_list = loop.run_until_complete(cls.scheduler.get_schedules())
+                for sch in schedule_list:
+                    # STARTUP type exclusion
+                    if int(sch.schedule_type) != 1:
+                        _logger.debug("{}-{}-{}-{}".format(str(sch.schedule_id), sch.name, int(sch.schedule_type), sch.process_name))
+                        # TODO: Errors on start up enable below line (if statement) otherwise it will queue the tasks for backup & restore (as inbuilt schedules)
```

Comment:
We should probably include backup at least, but maybe also restore. They may in the future add configuration. 

I've added these to FOGL-6799

Reply:
I am fixing one by one as there are issues in FOGL-6799 which needs to be fixed. Pushed changes into this branch. As this PR is created on top of FOGL-6799.